<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutSearch" stretch="0,0,1">
      <item>
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefresh">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitterExplorer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="indentation">
        <number>1</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">Title</string>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </column>
      </widget>
      <widget class="ImageView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddFile"/>
    <addaction name="actionAddDir"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionCreateDB"/>
    <addaction name="separator"/>
    <addaction name="actionLoadDB"/>
    <addaction name="actionUnloadDB"/>
    <addaction name="separator"/>
    <addaction name="actionCleanDB"/>
    <addaction name="actionCleanHashes"/>
    <addaction name="actionCleanPaths"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionThemeDarkMaroon"/>
    <addaction name="actionThemeDarkGreen"/>
    <addaction name="actionThemeDarkPurple"/>
    <addaction name="separator"/>
    <addaction name="actionThemeFusion"/>
    <addaction name="actionThemeWindows"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLoad"/>
   <addaction name="menuTheme"/>
  </widget>
  <action name="actionLoadDB">
   <property name="text">
    <string>Load DB</string>
   </property>
   <property name="statusTip">
    <string>Load a database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAddDir">
   <property name="text">
    <string>Add Directory</string>
   </property>
   <property name="statusTip">
    <string>Add all archives in a directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="text">
    <string>Add File</string>
   </property>
   <property name="statusTip">
    <string>Add a single archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exists the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionThemeDarkMaroon">
   <property name="text">
    <string>Dark Maroon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionThemeDarkGreen">
   <property name="text">
    <string>Dark Green</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionThemeDarkPurple">
   <property name="text">
    <string>Dark Purple</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionThemeFusion">
   <property name="text">
    <string>Fusion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionThemeWindows">
   <property name="text">
    <string>Windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionCreateDB">
   <property name="text">
    <string>Create DB</string>
   </property>
   <property name="statusTip">
    <string>Create a new database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionUnloadDB">
   <property name="text">
    <string>Unload DB</string>
   </property>
   <property name="statusTip">
    <string>Unload a database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionCleanDB">
   <property name="text">
    <string>Clean DB</string>
   </property>
   <property name="statusTip">
    <string>Removes entries with the same path but different hash from the database, keeping the correct entry</string>
   </property>
  </action>
  <action name="actionCleanHashes">
   <property name="text">
    <string>Clean hashes</string>
   </property>
   <property name="statusTip">
    <string>Removes all entries with the same hash from the database</string>
   </property>
  </action>
  <action name="actionCleanPaths">
   <property name="text">
    <string>Clean paths</string>
   </property>
   <property name="statusTip">
    <string>Removes all entries with the same path from the database</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>gui/imageview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
